apps/judicial-system/api/src/app/modules/file/file.controller.ts

```diff
@@ -100,7 +98,6 @@ export class FileController {
   @Header('Content-Type', 'application/pdf')
   async getRulingPdf(
     @Param('id') id: string,
-    @Query('useSigned', ParseBoolPipe) useSigned: boolean,
```

Comment:
The whole unsigned logic is no longer needed. Throwing out a lot of code.

Reply:
The old functionality that I just removed allowed the judge to preview the a ruling with changes in an already completed case. If the ruling had been signed, then the backend returned the signed ruling and not a new ruling generated with the changes just made. So, this logic was introduced to allow the client to ask for an unsigned generated copy of a ruling even if a signed copy existed in S3. This is no longer needed.

Reply:
Ok ok gotcha, thanks 🙏